packages/store-ui/src/atoms/Slider/README.md

```diff
@@ -0,0 +1,83 @@
+# Slider
+
+## Installation
+
+From the command line in your project directory, run npm install `@vtex/store-ui` or yarn add `@vtex/store-ui`.
+
+```cmd
+npm install @vtex/store-ui
+# or
+yarn add @vtex/store-ui
+```
+
+For style, you can install our default theme, run npm install `@vtex/theme-b2c-tailwind` or yarn add `@vtex/theme-b2c-tailwind`:
+
+```cmd
+npm install @vtex/theme-b2c-tailwind
+# or
+yarn add @vtex/theme-b2c-tailwind
+```
+
+```tsx
+import { Slider } from '@vtex/store-ui'
+import '@vtex/theme-b2c-tailwind/dist/index.css'
+```
```

Comment:
How about we add this whole initial part to `store-ui`'s README file, instead of this one inside of the component?

Reply:
I guess this would apply to all of the other components as well.

Reply:
Hmm, that makes sense!